src/BPP/BPP.java

```diff
@@ -1,12 +1,10 @@
 package BPP;
 import java.util.ArrayList;
 
-// bestFit() geeft je een int[] mee met de gewichten, returnt een arraylist met de gewichten gesorteerd in bakken: [[x, x, x], [x, x], [x]]
-
```

Comment:
nieuwe comment maken.

wat returned deze functie? een array met id's of een array met gewichten?